runtime/src/lib.rs

```diff
@@ -115,56 +78,33 @@ fn wait_for_cmd(_mbox: &mut Mailbox) {
     //}
 }
 
-fn handle_command(drivers: &mut Drivers) -> CaliptraResult<MboxStatusE> {
```

Comment:
Not sure I understand why this function needed to be split. Will probably make reviewing easier to not change this for now.

Reply:
That's fine I can re-combine this